.github/workflows/release-hf-container.yml

```
@@ -0,0 +1,120 @@
+      id-token: write
+      attestations: write
+
+    steps:
+    - name: 📥 Checkout repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: 🏷️ Set version
+      id: version
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:6:30: Double quote to prevent globbing and word splitting

(shellcheck)